### PR TITLE
Add `skip_typescript` to add string enums in `web-sys`

### PR DESCRIPTION
**Pull request by RunDevelopment**
Temporary solution for #4163.

I changed the webidl generator code and added a test case to see whether the fix works.

However, running the command from the `web-sys` README fails on my system, so I cannot re-generate the rs files.

```
PS C:\Users\micha\Git\wasm-bindgen> cargo run --release --package wasm-bindgen-webidl -- webidls src/features ./Cargo.toml
    Finished `release` profile [optimized] target(s) in 0.30s
     Running `target\release\wasm-bindgen-webidl.exe webidls src/features ./Cargo.toml`
Error: reading webidls directory

Caused by:
    Das System kann den angegebenen Pfad nicht finden. (os error 3)
error: process didn't exit successfully: `target\release\wasm-bindgen-webidl.exe webidls src/features ./Cargo.toml` (exit code: 1)
```

### CODE REVIEW

File: crates/cli/tests/reference.rs [hunk 1]
```diff
@@ -55,6 +55,14 @@ fn main() -> Result<()> {
...
     let contents = fs::read_to_string(test)?;
     let td = tempfile::TempDir::new()?;
+    let root = repo_root();
+    let root = root.display();
+
+    let dependencies = contents
+        .lines()
+        .filter_map(|l| l.strip_prefix("// DEPENDENCY: "))
+        .map(|l| "\n            ".to_string() + &l.trim().replace("{root}", &root.to_string()))
+        .fold(String::new(), |a, b| a + &b);
```
Comment (daxpedda) [inline crates/cli/tests/reference.rs:65]: This needs some documentation.

Comment (RunDevelopment) [reply crates/cli/tests/reference.rs:87]: I added docs.

File: crates/cli/tests/reference.rs [hunk 2]
```diff
@@ -124,7 +146,7 @@ fn runtest(test: &Path) -> Result<()> {
 }
 
 fn assert_same(output: &str, expected: &Path) -> Result<()> {
-    if env::var("BLESS").is_ok() {
+    if env::var("BLESS").is_ok() || true {
```
Comment (daxpedda) [inline crates/cli/tests/reference.rs:149]: Debugging leftover?

Comment (RunDevelopment) [reply crates/cli/tests/reference.rs:149]: Debugging leftover.

Comment (RunDevelopment) [reply crates/cli/tests/reference.rs]: Sorry about that.

**Comment by RunDevelopment** [comment]
And thank you for running the command for me.

**Comment by RunDevelopment** [comment]
We decided against this solution: https://github.com/rustwasm/wasm-bindgen/issues/4163#issuecomment-2405959257

**Pull request closed without merging**